frontend/src/metabase/visualizations/lib/utils.js

```diff
@@ -271,6 +271,10 @@ export function getDefaultDimensionAndMetric(series) {
   };
 }
 
+function isBreakoutDimensionCardinalitySensible(cardinality) {
+  return cardinality > 1 && cardinality < MAX_SERIES;
+}
```

Comment:
**1st Suggestion**
Do we want `cardinality >= 1` instead of `cardinality > 1`?

```suggestion
function isBreakoutDimensionCardinalitySensible(cardinality) {
  return cardinality >= 1 && cardinality > MAX_SERIES;
}
```

---
**2nd Suggestion**
Negating a `>` should lead to a `<=`. Was there a reason you changed from strictly greater than to strictly less than? 

`!(getColumnCardinality(cols, rows, cols.indexOf(dimensions[1])) > MAX_SERIES)` => `getColumnCardinality(cols, rows, cols.indexOf(dimensions[1])) <= MAX_SERIES`

```suggestion
function isBreakoutDimensionCardinalitySensible(cardinality) {
  return cardinality > 1 && cardinality <= MAX_SERIES;
}
```

---
**Combined**

```suggestion
function isBreakoutDimensionCardinalitySensible(cardinality) {
  return cardinality >= 1 && cardinality <= MAX_SERIES;
}
```